lib/src/layouts/yaru_master_detail_page.dart

```diff
@@ -55,7 +57,6 @@ class YaruMasterDetailPage extends StatefulWidget {
     this.layoutDelegate =
         const YaruMasterFixedPaneDelegate(paneWidth: _kDefaultPaneWidth),
     this.appBar,
-    this.initialIndex,
```

Comment:
same as `YaruCompactLayout` - we could keep this for convenience?

Reply:
:+1: